src/style.css

```diff
@@ -1,3 +1,70 @@
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
```

Comment:
Remember to use fallback fonts - alternative font-family in case the main one doesn't work. It's a good practice to specify a generic font family as a fallback in case the preferred font family is not available. For example, you could modify this line to: `font-family: Roboto, Arial, sans-serif;`